tests/cli/data/test_pofilter_listfilters/stdout.txt

```diff
@@ -48,6 +48,7 @@ tabs	Checks whether tabs are consistent between the two strings.
 unchanged	Checks whether a translation is basically identical to the original
         string.
 untranslated	Checks whether a string has been translated at all.
+untranslated_plurals	Check if any of the plural targets is translated while other isn't.
```

Comment:
Can we not just extend `untranslated` i.e. do we need a special test at all for this?